README.md

```diff
@@ -163,20 +178,20 @@ cd avod/data/outputs/avod_cars_example
 tensorboard --logdir logs
 ```
 
-Note: In addition to evaluating the loss, calculating accuracies, etc, the evaluator also runs the KITTI native evaluation code on each checkpoint. Predictions are converted to KITTI format and the AP is calculated for every checkpoint. The results are saved inside `scripts/offline_eval/results/avod_cars_example_results_0.1.txt` where `0.1` is the score threshold.
+Note: In addition to evaluating the loss, calculating accuracies, etc, the evaluator also runs the KITTI native evaluation code on each checkpoint. Predictions are converted to KITTI format and the AP is calculated for every checkpoint. The results are saved inside `scripts/offline_eval/results/avod_cars_example_results_0.1.txt` where `0.1` is the score threshold. IoUs are set to (0.7, 0.5, 0.5) 
```

Comment:
Do you wanna change these names here from `avod_cars_example_results_0.1.txt` -> `pyramid_cars_with_aug_example_results_0.1.txt` as well? 
Same for `cd avod/data/outputs/avod_cars_example` ->  `cd avod/data/outputs/pyramid_cars_with_aug_example`